src/iperf_api.c

```diff
@@ -32,7 +32,11 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#ifdef HAVE_GETOPT_H
 #include <getopt.h>
+#else
+#include "missing_getopt.h"
```

Comment:
Is it called `missing_getopt.h` upstream?  If not, I wouldn't bother with the name change since clashing isn't really a problem if (a) the file is non-existent and (b) we're talking about standard includes versus in-tree.